src/hip/gpu_get2e_subs.h

```diff
@@ -90,11 +90,15 @@ To understand the following comments better, please refer to Figure 2(b) and 2(d
  */
 #ifdef OSHELL
 #ifdef int_sp
-__global__ void 
-__launch_bounds__(HIP_SP_2E_THREADS_PER_BLOCK, 1) get_oshell_eri_kernel_sp()
+__global__ void
+__attribute__((amdgpu_waves_per_eu(HIP_SP_2E_WAVES_PER_CU,HIP_SPD_2E_WAVES_PER_CU)))
```

Comment:
Should this be twice `SP`? There is `SPD` as second argument to `amdgpu_waves_per_eu`.

Reply:
You are absolutely right. Good catch Andy!